spec/unit/transaction/report_spec.rb

```diff
@@ -404,6 +404,16 @@ def add_statuses(count, type = :file)
     expect(error_report.render).to validate_against('api/schemas/report.json')
   end
 
+  it "can make a round trip through yaml" do
+    report = generate_report
+
+    yaml_output = report.render(:yaml)
+    tripped = Puppet::Transaction::Report.convert_from(:yaml, yaml_output)
+
+    yaml_output.should =~ /^--- /
```

Comment:
I forgot this uses the deprecated rspec2 syntax which emits a deprecation warning. Could you change that to use rspec3 syntax:

```
    expect(yaml_output).to match(/^--- /)
```